src/instana/version.py

```diff
@@ -3,4 +3,4 @@
 
 # Module version file.  Used by setup.py and snapshot reporting.
 
-VERSION = "3.2.1"
+VERSION = "3.2.2"
```

Comment:
This new version releases new features, so it must be `3.3.0`. 
```suggestion
VERSION = "3.3.0"
```